db/builder.cc

```diff
@@ -173,12 +173,28 @@ Status BuildTable(
             prev_key.assign(keys.front());
             ok = ParseInternalKey(Slice(prev_key), &prev_ikey);
             assert(ok);
+
+            // The front item has the smallest seqno in the merge queue
```

Comment:
Hm, actually, line 131 might not catch all the keys, because MergeUntil calls Next()